Copy editing (#194)
openforis/sepal-doc: docs/source/setup/index.rst

Getting started with SEPAL
==========================

Set up an account and learn how to access the SEPAL interface

> ------------

Reviewer: ```suggestion
---------------------------------------------------------------
```

this is something you cannot see but when the documentation is built I see some parsing errors. 
Lines under the title need to be the same length

In this section of SEPAL documentation, you can learn about the key requirements needed to use the platform.

> This guide includes an Introduction to the SEPAL interface, as well as instructions that explain how you can :doc:`register`, Use Google Earth Engine (GEE) with SEPAL, :doc:`filezilla`, Use Norway’s International Climate & Forests Initiative (NICFI) - Planet Lab data, and Reset your password.

Reviewer: ```suggestion
This guide includes:
-   An Introduction to the SEPAL interface
-   Instructions that explain how you can :doc:`register`
-   Usage instructions for  Google Earth Engine (GEE) with SEPAL
-   Use of :doc:`filezilla` to download files
-   Use Norway’s International Climate & Forests Initiative (NICFI) - Planet Lab data
-   How to reset your password.
```

this is an example of how you can create a bullet point list. "-" and then 3 spaces, you can accept this suggestion and then modify the n text again if needed.
